examples/packer-azure-windows/Dockerfile.tmpl

```diff
@@ -0,0 +1,21 @@
+FROM quay.io/deis/lightweight-docker-go:v0.2.0
```

Comment:
nit: I vote for not using this as the base docker image as people external to our team are not familiar with it. If we can use maintained / stock base images, that would be preferred.

In Porter we have stopped using that base image and switched to 

```
FROM debian:stretch

ARG BUNDLE_DIR

RUN apt-get update && apt-get install -y ca-certificates
```

Argh! I just realized that I switched us over for the default template but not the `Dockerfile.tmpl`. I'm going to submit a PR to fix that now. 😊

Reply:
Ah, thanks for pointing this out.  That stock template isn't needed at this time, so I've just removed it.